src/Akeneo/Pim/Enrichment/Bundle/Elasticsearch/AbstractCursor.php

```diff
@@ -68,8 +69,11 @@ public function valid(): bool
      */
     public function count(): int
     {
-        if (null === $this->items) {
-            $this->rewind();
+        if (null === $this->count) {
+            $esQuery = $this->esQuery + ['track_total_hits' => true];
```

Comment:
```suggestion
            $esQuery = \array_replace($this->esQuery, ['track_total_hits' => true]);
```
(in case `track_total_hits` is already set to `false`, though it's very unlikely)